Sources/ProjectSpec/Scheme.swift

```diff
@@ -8,7 +8,7 @@ public typealias BuildType = XCScheme.BuildAction.Entry.BuildFor
 public struct Scheme: Equatable {
 
     public var name: String
-    public var build: Build
+    public var build: Build!
```

Comment:
This should of course should be `Build?`, but changing to a full optional led to many type-inference errors in `SpecLoadingTests` that I wasn't able to easily fix